app/src/main/java/com/sdpteam/connectout/qr_code/QRcodeEventActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.sdpteam.connectout.qr_code;
+
+import androidx.activity.result.ActivityResult;
+import androidx.activity.result.ActivityResultCallback;
+import androidx.activity.result.ActivityResultLauncher;
+import androidx.activity.result.contract.ActivityResultContracts;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import com.sdpteam.connectout.R;
+
+public class QRcodeEventActivity extends AppCompatActivity {
```

Comment:
From what I understood this activity is only used for playing around and for the demo. Maybe you should add a TODO to remind that this has to be removed soon